addons/mail/static/src/components/attachment_image/attachment_image.xml

```diff
@@ -3,28 +3,26 @@
     <t t-name="mail.AttachmentImage" owl="1">
         <div>
             <t t-if="attachmentImage and attachmentImage.attachment">
-                <div class="o_AttachmentImage d-flex o-details-overlay position-relative"
+                <div class="o_AttachmentImage d-flex position-relative"
                     t-att-class="{
-                        'o-composer-attachment': attachmentImage.attachment.isLinkedToComposer,
+                        'o-isAttachmentLinkedToComposer': attachmentImage.attachment.isLinkedToComposer,
                         'o-isUploading': attachmentImage.attachment.isUploading,
                     }"
                     t-att-title="attachmentImage.attachment.displayName ? attachmentImage.attachment.displayName : undefined"
                     t-att-data-id="attachmentImage.attachment.localId"
                     t-on-click="attachmentImage.onClickImage"
                     t-att-data-mimetype="attachmentImage.attachment.mimetype"
                 >
-                    <img class="img img-fluid" t-att-src="attachmentImage.imageUrl" t-att-alt="attachmentImage.attachment.name"/>
+                    <img class="o_AttachmentImage_image img img-fluid" t-att-src="attachmentImage.imageUrl" t-att-alt="attachmentImage.attachment.name"/>
                     <t t-if="attachmentImage.attachment.isUploading and attachmentImage.attachment.isLinkedToComposer">
-                        <div class="o_AttachmentImageUploading d-flex align-items-center justify-content-center position-absolute" title="Uploading">
-                            <i class="fa fa-spin fa-spinner"/>
+                        <div class="o_AttachmentImage_uploading d-flex align-items-center justify-content-center position-absolute" title="Uploading">
+                            <i class="fa fa-circle-o-notch fa-spinner"/>
```

Comment:
changed the modifier instead of the icon

Reply:
done